src/modules/Game.class.js

```diff
@@ -50,19 +223,135 @@ class Game {
    * `win` - the game is won;
    * `lose` - the game is lost
    */
-  getStatus() {}
+
+  getStatus() {
+    let emptyField = true;
+
+    for (let i = 0; i < 4; i++) {
+      for (let j = 0; j < 4; j++) {
+        if (this.field[i][j] === 2048) {
+          document.querySelector('.message-win').style.display = 'block';
+          document.removeEventListener('keydown', this.keyMove);
```

Comment:
You shouldn't do DOM manipulation inside of the Game class. Do all such things in main.js